CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Changed
+* **core + styling:** Tooltip ([#940](https://github.com/dso-toolkit/dso-toolkit/issues/940))
```

Comment:
Toch ook `dso-toolkit` zelf? We bieden de werkvorm immers nog steeds (ook) aan als CSS component